code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -106,10 +106,10 @@
 	name = "Fireball"
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
 
+/*
 /datum/spellbook_entry/rod_form
 	name = "Rod Form"
-	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form
-	cost = 40
+	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form */
```

Comment:
You again are misinterpreting my point and mindset. Please do not put words into my mouth. It would be appreciated if you toned down the hostility as I am not being hostile towards you. Regardless. I will go over your points.
Magic Missile-Nonlethal, better as a panic stun for bailing out. Does not down long enough to crit without looting stationside gear for use. Not Reliable.
Disintegrate-Does not leave corpses, melee only. Reliable but Unsuited for Artificer and Necromancer Builds
Fireball-Risk of hitting self, among the highest causes of wizard fatalities. Destroys Rwalls after a few shots. Semireliable.
Lightning bolt-needs to be charged before each use unless that has changed. Ambush situations are rarely a thing with wiz. Lightning Bolt/Tesla Blast are however Reliable assuming they are charged in advance. Point ceded for them.
Staff of Chaos-Instant Rejuv, Door Bolt No Effect, Teleport, are also all options. I do believe there are a couple more in it. Keep in mind that about 50% of the Polymorph bolts will make the target more dangerous. Not Reliable. The point being discussed is about Reliable ways to down.
Wands-Reliable but interferes with Artificer Build. Wands run out, Charge is an extra spell required for reliable wandbuild. Overall reliable, ceding this one.
Hulk Mode-Reliable, Melee Only. Ceding.
The Traps-Insufficient experience with spell, unable to provide counterpoints, seems useful for chases based on observation, less so for offense. 
Timestop-Does not drop target. Hilarious when coupled with Summon Lesser Guns or Arcane Blast. Not Reliable for point of discussion but still a fun spell.